Overview.bs

```diff
@@ -400,46 +401,43 @@ Note: The user agent is not required to run animations on every frame. It is leg
       provide a different service level according to their policy.
 
 
-When the user agent wants to <dfn>run animators</dfn>, it <em>must</em> iterates over <a>animator
-instance list</a> as |instance|:
+When the user agent wants to <dfn>run animators</dfn>, it <em>must</em> iterate over the values
+in the <a>(animator name, root element) to instance map</a> as |instance|, performing the following
+steps for each |instance|:
+
+  1. If |instance| is <b>null</b>, abort the following steps.
 
-  1. If the <a>animation requested flag</a> for the instance is <a>frame-current</a>
+  2. If the <a>animation requested flag</a> for the instance is <a>frame-current</a>
         or the proxies belonging to the |instance| will not be visible within the visual viewport
         of the current frame the user agent <em>may</em> abort all the following steps.
 
         Issue: Consider giving user agents permission to skip running animator instances to throttle
         slow animators.
 
-  2. Let |name| be the <a>animator name</a> of |instance|.
+  3. Let |name| be the <a>animator name</a> of |instance|.
 
-  3. Let |workletGlobalScope| be a {{AnimationWorkletGlobalScope}} from the list of <a>worklet's
+  4. Let |workletGlobalScope| be a {{AnimationWorkletGlobalScope}} from the list of <a>worklet's
         WorkletGlobalScopes</a> from the animation {{Worklet}}.
 
-  4. Let the |definition| be the result of looking up |name| on the |workletGlobalScope|'s
+  5. Let the |definition| be the result of looking up |name| on the |workletGlobalScope|'s
         <a>animator name to animator definition map</a>.
 
       If |definition| does not exist abort the following steps.
 
-  5. Let |animatorInstance| be the result of looking up |name| on |workletGlobalScope|'s
-        <a>animator name to instance map</a>. If |animatorInstance| is null run the following
-        substeps:
-
-        Issue: Todo: Add steps to create new animatorInstance given the animator definition.
```

Comment:
That would imply we need to iterate over something other than the animator instance map then. I only converted this wording from the previous iteration over the animator instance list, where even then the concept of checking whether the instance exists or not does not make sense.

We could instead iterate over the defined animator names and animator-root elements? That would actually allow us to consolidate creating animator instances to this set of steps?